proyecto prueba/Entitites/Bills.cs

```diff
@@ -0,0 +1,15 @@
+namespace proyecto_prueba.Entities
+{
+    public class Bills
+    {
+        public int BillNumber {get;set;}
```

Comment:
the code is more beauty if you put an space between get and set and brackets only when is a property.
public int BillNumber { get; set; }